packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -72,23 +70,48 @@ class _EditablePropertyItem extends StatelessWidget {
   final EditableArgument argument;
   final PropertyEditorController controller;
 
+  static const _hasArgIndicatorWidth = denseSpacing;
+
   @override
   Widget build(BuildContext context) {
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.start,
-      children: [
-        Flexible(
-          flex: 3,
-          child: Padding(
-            padding: const EdgeInsets.all(_PropertiesList.itemPadding),
-            child: _PropertyInput(argument: argument, controller: controller),
-          ),
+    final theme = Theme.of(context);
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          left:
+              argument.hasArgument
+                  ? BorderSide(
+                    color: theme.colorScheme.primary,
+                    width: _hasArgIndicatorWidth,
+                  )
+                  : BorderSide.none,
+          bottom: defaultBorderSide(Theme.of(context)),
         ),
-        if (argument.isRequired || argument.isDefault) ...[
-          Flexible(child: _PropertyLabels(argument: argument)),
-        ] else
-          const Spacer(),
-      ],
+      ),
+      child: _ListItemPadding(
+        additionalPadding: EdgeInsets.only(
+          left: argument.hasArgument ? noPadding : _hasArgIndicatorWidth,
+        ),
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
```

Comment:
nit: it might be simpler to just add a Container to this row directly:
```
Row(
  children: [
    Container(
      width: _hasArgIndicatorWidth,
      color: argument.hasArgument ? theme.colorScheme.primary : Colors.transparent,
    ),
    ...
  ],
),
```